File: Backend/Infrastructure/Data/Postgres/Entities/Transaction.cs
using Infrastructure.Data.Postgres.Entities.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.Data.Postgres.Entities;

public class Transaction : Entity<int> 
{
    public decimal Amount { get; set; }
    public string Description { get; set; }
    public int AccountID { get; set; }
    public Account Account { get; set; }
    public IList<UserTransactions> UserTransactions { get; set; }
}

File: Client/src/core/models/transaction.ts
import { Account } from "./account";
import { Entity } from "./entity.model";
import { UserTransactions } from "./usertransactions";

export class Transaction extends Entity<number> {
    amount?: number;
    description?: string;
    accountID?: number;
    account?: Account;
    userTransactions?: UserTransactions[];
}